django/santropolFeast/member/models.py

```diff
@@ -489,6 +489,13 @@ def notes(self):
         return self.client_notes.all()
 
     @property
+    def scheduled_statuses(self):
+        """
+        Returns scheduled status associated to this client
+        """
+        return self.client_scheduled_statuses.all()
+
```

Comment:
?? I don't understand, you simply ask me to rename a «method»..
OK, name is shorter, PEP8 compliant...
I already use this related name in a property method of Client model (l.492)